src/Views/CountrySelect.elm

```diff
@@ -0,0 +1,26 @@
+module Views.CountrySelect exposing (view)
+
+import Data.Country as Country exposing (Country)
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (..)
+
+
+view : List (Attribute msg) -> Country.Code -> (Country.Code -> msg) -> List Country -> Html msg
```

Comment:
It feels to me that these args could be passed as a `Config msg` record, as we often do with reusable views which accept a lot of configuration

Reply:
Sure, why not. Maybe feels a bit overengineered in this case with only one use case, but it'll be there for (if) we need it later ;)